execution_engine_testing/test_support/src/deploy_item_builder.rs

```diff
@@ -18,7 +18,7 @@ struct DeployItemData {
     pub session_code: Option<ExecutableDeployItem>,
     pub gas_price: u64,
     pub authorization_keys: BTreeSet<AccountHash>,
-    pub deploy_hash: DeployHash,
```

Comment:
So if this type was non-optional before, how was it NOT required? Was some default used?

Reply:
Couldn't we calculate `DeployHash` from the `DeployItemData`? What's missing?